fix tests that involve dates fail when OS culture has 24 hours format

mnieto opened the pull request:
# PR Summary

Inside the vsteam-lib.Test project, tests that have Asserts based on date format (i.e. `actual.CreatedOn.ToString("M/d/yyyy h:mm:ss tt").`) fails when ran in a Windows machine configured with 24h format, for example Spanish. 
See: https://stackoverflow.com/q/38976486/777551

Example of error:
![image](https://user-images.githubusercontent.com/7962206/208730583-767d9a9c-4e8f-4907-bbf5-f3b71ea9ee85.png)

The solution is to use an invariant culture so tests will be independent of the local configuration

## PR Checklist

- [x] [Write Help](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-help)
- [x] [Write Unit Test](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-unit-test)
- [x] [Update CHANGELOG.md](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#update-changelogmd)

mnieto committed:
fix tests that involve dates fail when OS culture has 24 hours format

mnieto committed:
update changelog

SebastianSchuetze commented on CHANGELOG.md:
```diff
@@ -12,6 +12,9 @@ Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/502) fr
 
+Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/506) from [Miguel Nieto](https://github.com/mnieto) the following:
+- fix tests that involve dates fail when OS culture has 24 hours format
+
```
Can you update to the latest (not released) version 7.12.0?

mnieto replied:
updated

mnieto committed:
Update CHANGELOG.md

Move merge 'fix tests that involve dates fail when OS culture has 24 hours format' to next unpublished version

mnieto committed:
Merge branch 'trunk' into FixTestWithDates

mnieto commented:
@SebastianSchuetze  can you please review and merge

mnieto commented:
@SebastianSchuetze, can you please review last changes and merge?
